### DISCUSSION TITLE
请问geosite的格式怎么写才对？

### DISCUSSION THREAD

**Discussion by calonye** [Openwrt-Passwall/openwrt-passwall2#602] [Q&A]
![image](https://github.com/xiaorouji/openwrt-passwall2/assets/45063293/6aa0f53b-6ca5-45b3-99a2-c2e529463b43)

我尝试 `geosite:***,***` 、 `geosite:***,换行`geosite:***`，或者上图，感觉都没生效，也没有任何地方可以看到有相关注释，我记得原本的规则是用逗号或者一行一个可以生效的。我试了下不行，并可能导致core不启动。

求教一下生效格式？我想写多个在其中一个分流规则里，不想建立多个分流规则

**Comment by adfnkjasd** [comment]
要geodata里有相关规则才可以，不是随便什么都可以的

**Reply by calonye** [reply]
主要是想知道格式是用逗号间隔还是用转行一行一个的

**Reply by adfnkjasd** [reply]
换行可以，逗号没试过

**Reply by adfnkjasd** [reply]
> 主要是想知道格式是用逗号间隔还是用转行一行一个的

另外，导致core不启用就是因为geodata里没有相关域名数据，这种原因导致的core不启用不管是逗号还是换行都不行

**Reply by calonye** [reply]
啊，这样子吗，我测试了单独另起一个规则又正常，比如，getosite:gfw 这个我放在一起，更行或者逗号都不行，另外新建一个规则，又没事~所以我才想不通··hhh
